Update edition-app.md (#686)
vtex-apps/io-documentation: docs/en/Concepts/edition-app.md

# Edition App

An Edition App consists of a bundle of settings, policies, back- and front-end apps encapsulated and exported by a [Sponsor Account](https://vtex.io/docs/concepts/sponsor-account/).

Its main objective is to facilitate the installation of numerous apps that might be considered as essential to set up one specific environment. 

![EditionApp](https://user-images.githubusercontent.com/60782333/91470034-927c0480-e86a-11ea-866e-54575f3c0975.png)

For example, consider the two Edition Apps developed by the `vtex` account:

> - The Edition Business (`vtex.edition-business@0.x`) - installs all the necessary apps to build a store with the [VTEX's CMS](https://help.vtex.com/tutorial/o-que-e-o-cms--EmO8u2WBj2W4MUQCS8262).

Reviewer: Eu substituiria: 

> the [VTEX's CMS](https://help.vtex.com/tutorial/o-que-e-o-cms--EmO8u2WBj2W4MUQCS8262)

Por: 

> our legacy [CMS](https://help.vtex.com/en/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/6OCY6S9tqBXPD5mgpbBInC)

A palavra *CMS* na verdade se refere a tudo que é possível fazer num admin para gerenciar conteúdo. [Site Editor](https://storecomponents.myvtex.com/admin/cms/site-editor) e [Pages](https://storecomponents.myvtex.com/admin/cms/pages) fazem parte do CMS de lojas no Store Framework.

Reviewer: Note que eu mudei o link da doc do CMS

> - The Edition Business (`vtex.edition-business@0.x`) - installs all the necessary apps to build a store with the [VTEX's CMS](https://help.vtex.com/tutorial/o-que-e-o-cms--EmO8u2WBj2W4MUQCS8262).

Author: ```suggestion
- The Edition Business (`vtex.edition-business@0.x`) - installs all the necessary apps to build a store with our legacy [CMS](https://help.vtex.com/en/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/6OCY6S9tqBXPD5mgpbBInC).
```

> - The Edition Store (`vtex.edition-store@2.x`) - installs all the necessary apps to develop a store with the [VTEX IO's Store Framework](https://vtex.io/docs/getting-started/build-stores-with-store-framework/1/).

Reviewer: Eu substituiria:

> the [VTEX IO's Store Framework]

Por:

> [Store Framework]

Podemos simplificar dentro da documentação. Se a pessoa já tá na documentação do VTEX IO, ela já vai ter esse contexto. Mesmo que não tenha isso formado, o Getting Started apresenta isso.

> - The Edition Store (`vtex.edition-store@2.x`) - installs all the necessary apps to develop a store with the [VTEX IO's Store Framework](https://vtex.io/docs/getting-started/build-stores-with-store-framework/1/).

Author: ```suggestion
- The Edition Store (`vtex.edition-store@2.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://vtex.io/docs/getting-started/build-stores-with-store-framework/1/).
```

> Also, not only `vtex`, but any VTEX account that meets the [requirements needed to be a Sponsor Account](https://vtex.io/docs/recipes/development/becoming-a-sponsor-account/) can develop and release its own Edition App.

Reviewer: Eu substituiria:

> Also, not only `vtex`, but any VTEX account

Por:

> In fact, any VTEX account

Reviewer: Também colocaria "their own Edition Apps", no plural, porque não precisa ser só uma.

> Also, not only `vtex`, but any VTEX account that meets the [requirements needed to be a Sponsor Account](https://vtex.io/docs/recipes/development/becoming-a-sponsor-account/) can develop and release its own Edition App.

Author: ```suggestion
In fact, any VTEX account that meets the [requirements needed to be a Sponsor Account](https://vtex.io/docs/recipes/development/becoming-a-sponsor-account/) can develop and release their own Edition Apps.
```

> Hence, besides the fundamental development environment set up by a `vtex` native Edition App, it's possible to extend an Edition App by creating a new and customized one that meets your account family needs. 
> 
> Notice that this is especially useful for complex account families, such as the ones under the same brand or holding.

Author: ```suggestion
Notice that this can be especially useful for complex account families, such as the ones under the same brand or holding.

That's because, besides the fundamental development environment set up by a `vtex` native Edition App, a Sponsor Account can extend an Edition App by creating a new and customized one that meets its account family needs. 
```

> Notice that this is especially useful for complex account families, such as the ones under the same brand or holding.

Reviewer: Eu achei estranho ler "account family needs" na linha 26, a frase da linha 28 esclareceu bastante. Eu inverteria a ordem das frases (ajustando o que fosse necessário pra fazer sentido)

<div class="alert alert-info">

> Given its importance and considering that its misuse could lead to critical situations, it's always necessary to <a href ="https://help-tickets.vtex.com/smartlink/sso/login/zendesk">open a ticket</a> to the VTEX Support Team in cases you want to change the Edition App installed in an account.

Author: ```suggestion
To change the Edition App installed in an account, you must <a href ="https://help-tickets.vtex.com/smartlink/sso/login/zendesk">open a support ticket</a>. This is required to avoid critical issues that could be caused by its misconfiguration.
```

> Given its importance and considering that its misuse could lead to critical situations, it's always necessary to <a href ="https://help-tickets.vtex.com/smartlink/sso/login/zendesk">open a ticket</a> to the VTEX Support Team in cases you want to change the Edition App installed in an account.

Reviewer: Eu justificaria depois, pra dar mais destaque pra informação central:

> To change the Edition App installed in an account, you must <a href ="https://help-tickets.vtex.com/smartlink/sso/login/zendesk">open a support ticket</a>. This is required to avoid critical issues that could be caused by its misconfiguration.

</div>

> It's important to be aware that, despite the inherited apps - such as the ones from the `vtex` Editions, an Edition App can only contain apps exclusively developed by the same account responsible for the development of that Edition.

Author: ```suggestion
Note that all apps must have the same vendor as the Edition App that bundles them. The only exceptions are inherited apps - those that are included as part of the dependencies of the Edition app.
```

> It's important to be aware that, despite the inherited apps - such as the ones from the `vtex` Editions, an Edition App can only contain apps exclusively developed by the same account responsible for the development of that Edition.

Reviewer: Mais um cenário em que inverter a ordem ajudaria a dar mais clareza para o texto:

> Note that all apps must have the same vendor as the Edition App that bundles them. The only exception are inherited apps - those that are included as part of the dependencies of the Edition app.

Usei [isso aqui](https://vtex.slack.com/archives/C1Q399J9H/p1598360736049600?thread_ts=1598358006.016500&cid=C1Q399J9H) como referência para esclarecer o que significa "inherited".

Reviewer: Acho que vale confirmar se toda Edition precisa ter uma Edition da VTEX como dependência direta ou indireta. Me parece que sim, daí isso daria mais uma frase:

> All Editions apps must depend, directly or indirectly, on either `vtex.edition-business` or `vtex.edition-store`.

> Also, keep in mind that, once exported, the apps and configurations that an Edition App includes cannot be changed by the sponsored accounts that had it installed. That means that only the Sponsor Account is authorized to perform modifications in the apps exported by its Edition App, but requiring, in the following, a new release of its Edition.

Author: ```suggestion
Keep in mind that Sponsored Accounts cannot change the apps and configurations installed through an Edition App. Only the Sponsor Account is authorized to do that, by releasing a new version of its Edition App and requiring it to be reinstalled.
```

> Also, keep in mind that, once exported, the apps and configurations that an Edition App includes cannot be changed by the sponsored accounts that had it installed. That means that only the Sponsor Account is authorized to perform modifications in the apps exported by its Edition App, but requiring, in the following, a new release of its Edition.

Reviewer: Tentando simplificar:

> Keep in mind that Sponsored Accounts cannot change the apps and configurations they inherit from their Sponsor Accounts through Edition apps. Only the Sponsor Account is authorized to do that, by releasing a new version of its Edition app.

> Summing up, by encapsulating all the necessary settings, policies, back-, and front-end apps, an Edition App provides all the essential functionalities to set up one desired environment.

Author: ```suggestion
To sum it up, by encapsulating all the necessary settings, policies, back-, and front-end apps, an Edition App provides all the essential functionalities to set up one desired environment.
```

> Summing up, by encapsulating all the necessary settings, policies, back-, and front-end apps, an Edition App provides all the essential functionalities to set up one desired environment.

Reviewer: "Summing up" me parece estranho. "To sum it up" seria o que eu usaria, de repente vale fazer uma pesquisa pra ver se esse formato é bastante utilizado.

- The Edition Store (`vtex.edition-store@2.x`) - installs all the necessary apps to develop a store with the [Store Framework](https://vtex.io/docs/getting-started/build-stores-with-store-framework/1/).

In fact, any VTEX account that meets the [requirements needed to be a Sponsor Account](https://vtex.io/docs/recipes/development/becoming-a-sponsor-account/) can develop and release their own Edition Apps.

Notice that this can be especially useful for complex account families, such as the ones under the same brand or holding.

That's because, besides the fundamental development environment set up by a `vtex` native Edition App, a Sponsor Account can extend an Edition App by creating a new and customized one that meets its account family needs. 

<div class="alert alert-info">
To change the Edition App installed in an account, you must <a href ="https://help-tickets.vtex.com/smartlink/sso/login/zendesk">open a support ticket</a>. This is required to avoid critical issues that could be caused by its misconfiguration.
</div>

> Note that all apps must have the same vendor as the Edition App that bundles them. The only exceptions are inherited apps - those that are included as part of the dependencies of the Edition app.
> 
> All Editions apps must depend, directly or indirectly, on either `vtex.edition-business` or `vtex.edition-store`.

Author: ```suggestion
Note that all Edition apps must depend, directly or indirectly, on either `vtex.edition-business` or `vtex.edition-store`.

The apps that are part of the Edition App included as a dependency are considered inherited apps.

Despite these, all apps of an Edition App must have the same vendor as the Edition App that bundles them.
```

# Edition App

> An Edition App consists of a bundle of settings, policies, back- and front-end apps encapsulated and exported by a [Sponsor Account](https://vtex.io/docs/concepts/sponsor-account/).

Reviewer: Aqui eu escolheria entre "back" ou "back-end", estranhei o uso do hífen solto

> An Edition App consists of a bundle of settings, policies, back- and front-end apps encapsulated and exported by a [Sponsor Account](https://vtex.io/docs/concepts/sponsor-account/).

Author: ```suggestion
An Edition App consists of a bundle of settings, policies, back-end and front-end apps encapsulated and exported by a [Sponsor Account](https://vtex.io/docs/concepts/sponsor-account/).
```
